miso-web/src/main/webapp/scripts/list_run.js

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2012. The Genome Analysis Centre, Norwich, UK
+ * MISO project contacts: Robert Davey @ TGAC
+ * *********************************************************************
+ *
+ * This file is part of MISO.
+ *
+ * MISO is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * MISO is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with MISO.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * *********************************************************************
+ */
+
+ListTarget.run = {
+  name : "Runs",
+  createUrl : function(config, projectId) {
+    if (projectId) {
+      return "/miso/rest/run/dt/project/" + projectId;
+    } else if (config.sequencer) {
+      return "/miso/rest/run/dt/sequencer/" + config.sequencer;
+    } else if (config.platformType) {
+      return "/miso/rest/run/dt/platform/" + config.platformType;
+    } else {
+      return "/miso/rest/run/dt";
+    }
+  },
+  createBulkActions : function(config, projectId) {
+    return [];
+  },
+  createStaticActions : function(config, projectId) {
+    return (!projectId && config.platformType) ? [ {
+      name : "Add",
+      handler : function() {
+        window.location = '/miso/run/new/' + config.platformType;
```

Comment:
Could you update the button text to include the platformType? It's not immediately obvious that the add button changes depending on the tab.